usaspending_api/awards/models_matviews.py

```diff
@@ -437,6 +437,27 @@ class Meta:
         db_table = 'summary_transaction_geo_view'
 
 
+class SummaryStateView(models.Model):
```

Comment:
Is the FIPS code not needed here?

Reply:
Unless I am missing anything, State Code (CA, VA, WV) is sufficient for querying and grouping transactions. The business logic wrapping the queries to this matview handle the validation and conversation between fips to state code.